tests/prefetch_tests.cpp

```diff
@@ -53,7 +53,7 @@ struct PrefetchTest : public ::testing::Test {
     // https://docs.nvidia.com/cuda/cuda-runtime-api/group__CUDART__MEMORY.html#group__CUDART__MEMORY_1g8048f6ea5ad77917444567656c140c5a
     // specifically for when cudaMemRangeAttribute::cudaMemRangeAttributeLastPrefetchLocation is
     // used.
-    constexpr size_t prefetch_data_size = 4;
+    [[maybe_unused]] constexpr size_t prefetch_data_size = 4;
```

Comment:
Could this just be inside of the if statement instead so that it will only be defined if it is used?

Reply:
Yes it can. Updated.